packages/iris-grid/src/IrisGridTableModel.js

```diff
@@ -516,16 +517,25 @@ class IrisGridTableModel extends IrisGridModel {
   get columns() {
     return this.getColumnsWithLayoutHints(
       this.table.columns,
-      this.table.layoutHints
+      this.layoutHints,
+      this.frozenColumns
     );
   }
 
+  get frozenColumns() {
+    if (this.userFrozenColumns && this.userFrozenColumns != null) {
```

Comment:
Only need the != null check:

```suggestion
    if (this.userFrozenColumns != null) {
```

Reply:
Could even just do `this.userFrozenColumns ?? this.layoutHints?.frozenColumns`... but see my other comment about memoizing all these functions.
Right now if there are no hints, we're getting a new frozenColumns array on each call, which causes the `get columns()` call to not memoize correctly, which could cause many unnecessary updates/renders.